docs/oauth/how-clerk-implements-oauth.mdx

```diff
@@ -0,0 +1,130 @@
+| `public_metadata` | Grant access to the user's public and unsafe metadata |
+| `private_metadata` | Grant access to the user's private metadata |
+| `openid` | Enables the OpenID Connect flow |
+
+> [!NOTE]
+> Support for adding custom OAuth scopes is **not yet available**, but development is underway. The goal is to allow custom scopes to be added, accepted, and checked through Clerk SDKs. An update will be provided when this feature is available.
+>
+> For early access to custom OAuth scopes, please vote or provide feedback on the [roadmap here](https://feedback.clerk.com/roadmap?id=d2d88be9-4d4f-45e6-997e-61d0b2a34bc9).
+
+### Token expiration and management
+
+- OAuth access tokens expire after 1 day.
+- Refresh tokens **never expire**.
+- Authorization codes expire after 10 minutes.
+- OIDC `id_token`s expire after 1 day.
+
+### Authorization server metadata
+
+An important feature of modern OAuth 2.0 and OpenID Connect (OIDC) implementations is **authorization server metadata**. This is a standardized JSON document published by the authorization server that describes its configuration, supported features and endpoints. This metadata makes it easier for clients to automatically discover important endpoints and features without needing manual configuration.
+
+You can easily retrieve this metadata by loading your **Frontend API URL** (FAPI) with `/.well-known/oauth-authorization-server` appended to it (e.g. `https://verb-noun-00.clerk.accounts.dev/.well-known/oauth-authorization-server` for a development environment or `https://clerk.<INSERT_YOUR_APP_DOMAIN>.com/.well-known/oauth-authorization-server` for production). Your **Frontend API URL** can be found on the [**API keys**](https://dashboard.clerk.com/last-active?path=api-keys) page in the Clerk Dashboard.
+
+This endpoint exposes all relevant details in a standardized JSON format.
+
+A sample metadata document might look like this:
+
+```json
+{
+  "issuer": "https://well-hagfish-71.clerk.accounts.dev",
+  "authorization_endpoint": "https://well-hagfish-71.clerk.accounts.dev/oauth/authorize",
+  "token_endpoint": "https://well-hagfish-71.clerk.accounts.dev/oauth/token",
+  "jwks_uri": "https://well-hagfish-71.clerk.accounts.dev/.well-known/jwks.json",
+  "response_types_supported": ["code"],
+  "grant_types_supported": ["authorization_code", "refresh_token"],
+  "token_endpoint_auth_methods_supported": ["client_secret_basic", "none"],
+  "scopes_supported": ["openid", "profile", "email", "public_metadata", "private_metadata"],
+  "subject_types_supported": ["public"],
+  "id_token_signing_alg_values_supported": ["RS256"],
+  "claims_supported": ["sub", "iss", "aud", "exp", "iat", "email", "name"],
+  "service_documentation": "https://clerk.com/docs",
+  "ui_locales_supported": ["en"],
+  "op_tos_uri": "https://clerk.com/legal/standard-terms",
+  "code_challenge_methods_supported": ["S256"]
+}
+```
+
+### Building a Model Context Protocol (MCP) service using Clerk’s OAuth server
```

Comment:
Just a note, once we run this release and have per-sdk guides, I think this will be a good candidate for lifting into its own page rather than a section